src/pages/IndividualArticle.jsx

```diff
@@ -74,6 +99,12 @@ export default function IndividualArticle () {
         </div>
         <section className="indent-4 py-4 px-2">{articleData.body}</section>
         <section className="indent-4 p-2">Comments: </section>
+        <form onSubmit={handleSubmit}>
+            <p><label htmlFor="commentBox">Post a comment as: {username}</label></p>
+            <textarea required id="commentBox" rows="4" cols="50"></textarea>
```

Comment:
This is **not** a controlled component. Please refactor this form so it is controlled https://notes.northcoders.com/courses/js-front-end/controlled-components

Reply:
https://react.dev/reference/react-dom/components/textarea#controlling-a-text-area-with-a-state-variable